tools/ci_build/github/azure-pipelines/templates/c-api-linux-cpu.yml

```diff
@@ -74,11 +93,10 @@ jobs:
           arch: '${{parameters.OnnxruntimeNodejsBindingArch}}'
           os: 'linux'
           artifactName: 'drop-onnxruntime-nodejs-linux-${{parameters.OnnxruntimeArch}}'
-
-    - template: component-governance-component-detection-steps.yml
-      parameters :
-        condition : 'succeeded'
-
+    - ${{ if not(eq(parameters.OnnxruntimeNodejsBindingArch, 'arm64')) }}:
```

Comment:
Is it enough? By default they would inject such a task to every our build job.

Reply:
I also set the 
  variables:
  - name: skipComponentGovernanceDetection
    value: ${{eq('${{parameters.OnnxruntimeNodejsBindingArch}}', 'arm64')}}